targets/targets.json

```diff
@@ -582,10 +582,13 @@
         "macros": ["CPU_MKW24D512VHA5", "FSL_RTOS_MBED"],
         "inherits": ["Target"],
         "detect_code": ["0250"],
-        "device_has": ["USTICKER", "LPTICKER", "RTC", "ANALOGIN", "I2C", "I2CSLAVE", "INTERRUPTIN", "PORTIN", "PORTINOUT", "PORTOUT", "PWMOUT", "SERIAL", "SERIAL_FC", "SLEEP", "SPI", "SPISLAVE", "STDIO_MESSAGES", "TRNG", "FLASH"],
+        "device_has": ["USTICKER", "LPTICKER", "RTC", "ANALOGIN", "I2C", "I2CSLAVE", "INTERRUPTIN", "PORTIN", "PORTINOUT", "PORTOUT", "PWMOUT", "SERIAL", "SERIAL_FC", "SLEEP", "SPI", "SPISLAVE", "STDIO_MESSAGES", "TRNG", "FLASH", "802_15_4_PHY", "MCR20A"],
```

Comment:
On this, I think (but am not totally certain) that "MCR20A" should be in "extra_labels". That seems to be where specific parts are named, whereas "device_has" is for general technologies.

Reply:
Hmm.. `extra_labels` means that it would pick up folder like `TARGET_MCR20A` but it would not be available as a build time flag.

`device_has` is a build time macro, but not visible in the `mbed_config.h`

Reply:
@sg- suggests that this - driver providing an implementation - is the intended use of the new `component`, and that should work both ways. Worth a try?